<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_exportResult</class>
 <widget class="QDialog" name="Dialog_exportResult">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>312</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>541</width>
    <height>312</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>541</width>
    <height>317</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>541</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export results</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox_fileFormat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>File format</string>
   </property>
   <widget class="QRadioButton" name="radioButton_ascii">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ascii</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_json">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>json</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_includeProfilesCurvesData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>311</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>include profile curve data</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Boxes">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Boxes</string>
   </property>
   <widget class="QRadioButton" name="radioButton_allBoxes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>All boxes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_onlyCurrentBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Only currently selected box (#&lt;...&gt;)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>269</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectOutputFileAndExport">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>269</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Select output file and export</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_flaggedProfil">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>521</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>For flagged profile</string>
   </property>
   <widget class="QCheckBox" name="checkBox_setLinRegResultAsEmpty">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>501</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set offset measurement and linear regression parameters as empty</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_setProfileCurveDataAsEmpty">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set profile curve data as empty</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>radioButton_ascii</tabstop>
  <tabstop>radioButton_json</tabstop>
  <tabstop>radioButton_allBoxes</tabstop>
  <tabstop>radioButton_onlyCurrentBox</tabstop>
  <tabstop>checkBox_includeProfilesCurvesData</tabstop>
  <tabstop>checkBox_setLinRegResultAsEmpty</tabstop>
  <tabstop>checkBox_setProfileCurveDataAsEmpty</tabstop>
  <tabstop>pushButton_selectOutputFileAndExport</tabstop>
  <tabstop>pushButton_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
